server/api/submission_handler.py

```diff
@@ -0,0 +1,35 @@
+from flask import jsonify, Blueprint, request
+from api import db
+from models import Contest, Submission, User
+from datetime import date
+submission_handler = Blueprint('submission_new_handler', __name__)
+
+
+@submission_handler.route('/contestImage/submission/<contest_id>/<user_id>', methods=['POST'])
+def create_submission(contest_id, user_id):
+    try:
+        contest = Contest.query.filter_by(id=contest_id)
+        if not contest:
+            raise Exception
+    except Exception:
+        return jsonify({'notFound': 'Contest not found'})
+
+    submission_user = User.query.filter_by(id=user_id)
+
+    if request.method == 'POST':
+        data = request.form
+        new_submission = Submission(contest_id=contest.id, submiter_id=submission_user.id, active=data.get('status'),
```

Comment:
since you're not utilizing this model in any way, you don't need to initialize `submission_user`, you can just pass in `user_id`

unless you want to verify that the user exists before hand